src/core/error_handling.h

```diff
@@ -7,13 +7,17 @@
 #include <emscripten.h>
 
 typedef int errcode;
+#include "hiwire.h"
 
 int
 error_handling_init();
 
 errcode
 log_error(char* msg);
 
+errcode
```

Comment:
Can you add a comment indicating that this is supposed to be not used in any existing code?

Reply:
You mean like something like "This is a debug tool, it's probably dead code but please don't remove it"?

Reply:
Yeah. Explain that it is intentionally dead, not accidentally dea.

Reply:
Okay, will do.